src/ngx-smart-modal/src/components/ngx-smart-modal.component.ts

```diff
@@ -183,7 +186,7 @@ export class NgxSmartModalComponent implements OnInit, OnDestroy {
         me.onAnyCloseEventFinished.emit(me);
       }, this.hideDelay);
 
-      window.removeEventListener('keyup', this.escapeKeyboardEvent);
+      this.isBrowser() && window.removeEventListener('keyup', this.escapeKeyboardEvent);
```

Comment:
For all your browser tests, it is preferable to detail statements to improve readability as : 
```
if (this.isBrowser ) {
 // do some stuff
}
```

Reply:
Done for all browser tests.